proto/foodfinder.proto

```diff
@@ -77,4 +78,7 @@ message Item {
   string name = 1;
   float price = 2;
   uint32 quantity = 3;
-}
\ No newline at end of file
+}
+
+// Empty message for one way grpc calls 
+message Empty {}
```

Comment:
Just FYI: There is a pre-defined message type `google.protobuf.Empty`. To use it, just `import "google/protobuf/empty.proto";` that the beginning of the proto file, and `using google::protobuf::Empty;` in your cpp files.

Reply:
I saw that before but couldn't get the import working so I just created one of my own. I took a look at how you did it and finally got it working - thanks,